data/json/mapgen/prison_1.json

```diff
@@ -259,89 +259,152 @@
+        "````~~~~~~~~~     *  % F                       F                  |,,,,B,,,,,B,,,,|,,,,B,,,,,B,,,T|  F %        ~~~~````",
+        "```~~~~~~~~~~     ** % F                       F                  |,,,,G,,,,,G,,,,|,,,,G,,,,,G,,,,|  F %        ~~~~````",
+        "``````~~~~~~    ***  % F                       F                  |,,,,B,,,,,B,,,,|,,,,B,,,,,B,,bb|  F %        ~~~`````",
+        "```~~~~~~~~    **    % F                       F                  |----|,,,,,|----|----|,,,,,|----|  F %        ~~~~````",
+        "`````~~~~~    **     % F                       F                  |,,,,B,,,,,B,,,,|,,,,B,,,,,B,,,T|  F %       ~~~~`````",
+        "```~~~~~~    **      % F                       F                  |,,,,G,,,,,G,,,,|,,,,G,,,,,G,,,,|  F %      ~~~~~`````",
+        "```~~~~~~ *****      % F                       F                  |,,,,B,,,,,B,,,,|,,,,B,,,,,B,,bb|  F %      ~~~~~`````",
+        "`~~~~~~~  *          % F                       Fffffffffffffffffff|----|-----|----|----|-----|----|  F %    ~~~~~~``````",
+        "````~~~~***          % F                                                                             F %  ~~~~~~````````",
+        "```~~~~~~ *        |--+--|                                                                         |--+--|~~~~~~````````",
+        "```~~~~~~**        |,,,,,|fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff|,,,,,|~~~~~`````````",
+        "````~~~~***        |,,,,,+                                                                         +,,,,,|~~~```````````",
+        "```~~~~***~~~~~    |,,,,,|%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%|,,,,,|~~~```````````",
+        "```~~~~~**~~~~~~~~ |-----|                                                                         |-----|~~~```````````",
+        "`````````*~~~~~~~~~~~~~~~                                                                        ~~~~~~~~~~`````````````",
+        "```````***``~~~~~~~~~~~~~~~~~                                                               ~~~~~~``````````````````````",
+        "```````**```````````````~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~```````````````````````````",
+        "````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````"
       ],
       "palettes": [ "prison_palette" ],
-      "terrain": { "~": [ "t_water_sh" ], "`": [ "t_water_dp" ] }
+      "terrain": {
+        "~": [ "t_water_sh" ],
+        "*": [ "t_dirt" ],
+        "!": [ "t_dock" ],
+        "^": [ "t_concrete" ],
+        "$": [ "t_brick_wall" ],
+        "?": [ "t_door_locked" ],
+        "(": [ "t_pavement_y" ],
+        ")": [ "t_guardrail" ],
+        "1": [ "t_window_bars_alarm" ],
+        "q": [ "t_floor" ],
+        "A": [ "t_floor" ],
+        "`": [
```

Comment:
For these lists, you can do `[ [ "t_water_dp", 20 ], "t_water_sh" ]` since #28542.

Reply:
Oh, shoot, I totally forgot about this. Thanks!